tools/mtouch/mtouch.csproj

```diff
@@ -202,9 +202,6 @@
     <Compile Include="..\linker\ObjCExtensions.cs">
       <Link>Xamarin.Linker\ObjCExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\linker\RemoveSelectors.cs">
```

Comment:
One of my usual mistakes :-) but in this case *no*, the pattern is still used on XM so the step will continue to run with `mmp`